app/models/form/lettings/questions/tenancy_type.rb

```diff
@@ -0,0 +1,37 @@
+class Form::Lettings::Questions::TenancyType < ::Form::Question
```

Comment:
This is the file with the changes from the ticket in them.
It is not really a new file it is renamed from the above "deleted" file tenancy.rb